webapp/src/components/viewHeader/__snapshots__/filterComponent.test.tsx.snap

```diff
@@ -1013,7 +1013,7 @@ exports[`components/viewHeader/filterComponent return filterComponent and filter
             type="button"
           >
             <span>
-              includes
+              (unknown)
```

Comment:
@Rajat-Dabade is this an expected side effect of the change?

Reply:
@Pinjasaur, Yes this is expected.